rule.go

```diff
@@ -70,3 +71,25 @@ func (r RuleSet) RegisteredFor(n ast.Node) []Rule {
 func (r RuleSet) IsRuleSuppressed(ruleID string) bool {
 	return r.RuleSuppressedMap[ruleID]
 }
+
+// TODO: move to new file? If so what should be the name of the file?
+type AnalyzerSet struct {
```

Comment:
Yeah, maybe move it into `analyzers_set.go` in the `analyzers` package.

Reply:
Done